### PR TITLE
Add external tutorials data and refactor tutorials

### PR DESCRIPTION
**Pull request by minimalsm**
## Description
Adds external tutorial data 

### Todo
- [ ] Refactor each tutorial tag to its own individual component
- [ ] Refactor metadata

### CODE REVIEW

**Review by KeKo6988 — APPROVED** [review]

**Review by KeKo6988 — APPROVED** [review]

**Review by KeKo6988 — APPROVED** [review]

**Review by wackerow — APPROVED** [review]
lgtm! @minimalsm thanks for bringing in these other tutorials

**Comment by wackerow** [comment]
@minimalsm Are these "todos" you have listed intended for this PR, or were you planning to break these off?

**Comment by minimalsm** [comment]
@wackerow this can be merged as is and I'll put up a PR for the todos this week.

File: src/data/tutorials.json [hunk 1]
```diff
@@ -1,14 +1,69 @@
...
+    "description": "Learn how to fetch the current price of Bitcoin, Ethereum and other cryptocurrencies in your Solidity smart contracts.",
+    "author": "Harry Papacharissiou",
+    "authorGithub": "https://github.com/kwsantiago",
+    "tags": ["solidity", "oracles"],
+    "skillLevel": "beginner",
+    "lang": "en",
+    "publishDate": "01/05/2021"
+  },
+  {
+    "url": "https://blog.chain.link/fetch-current-crypto-price-data-solidity/",
```
Comment (samajammin) [inline src/data/tutorials.json:59]: Looks like a duplicate here?

**Pull request merged**